RecoTracker/Configuration/python/RecoTracker_cff.py

```diff
@@ -38,5 +38,6 @@
 trackingGlobalReco = cms.Sequence(trackingGlobalRecoTask)
 
 from Configuration.Eras.Modifier_fastSim_cff import fastSim
-_fastSim_trackingGlobalReco = cms.Sequence(trackExtrapolator)
+ckftracks_fast = cms.Sequence(doAlldEdXEstimators)
+_fastSim_trackingGlobalReco = cms.Sequence(ckftracks_fast*trackExtrapolator)
 fastSim.toReplaceWith(trackingGlobalReco,_fastSim_trackingGlobalReco)
```

Comment:
How about the following?
```python
fastSim.toReplaceWith(trackingGlobalRecoTask, cms.Task(doAlldEdXEstimators, trackExtrapolator))
```

Reply:
It actually needs to be `doAlldEdXEstimatorsTask` since sequences can't be added to tasks